api/cmd/stop.go

```diff
@@ -1,54 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package cmd
-
-import (
-	"fmt"
-	"os"
-	"syscall"
-
-	"github.com/spf13/cobra"
-
-	"github.com/apisix/manager-api/internal/conf"
-	"github.com/apisix/manager-api/internal/utils"
-)
-
-func newStopCommand() *cobra.Command {
-	return &cobra.Command{
-		Use:   "stop",
```

Comment:
wow, We should keep the `stop` command, it has nothing to do with the daemon.

Reply:
@nic-chen The original code looked for the specified process by the PID file, which proved unreliable by many bug reports, and this part has been removed. Therefore, the original `stop` operation is not well supported.